Libs/DICOM/Widgets/ctkDICOMQueryRetrieveWidget.cpp

```diff
@@ -190,10 +190,10 @@ void ctkDICOMQueryRetrieveWidget::query()
     ctkDICOMQuery* query = new ctkDICOMQuery;
     d->CurrentQuery = query;
     query->setCallingAETitle(d->ServerNodeWidget->callingAETitle());
-    query->setCalledAETitle(parameters["AETitle"].toString());
-    query->setHost(parameters["Address"].toString());
-    query->setPort(parameters["Port"].toInt());
-    query->setPreferCGET(parameters["CGET"].toBool());
+    query->setCalledAETitle(parameters[QApplication::translate("ctkDICOMServerNodeWidget", "AETitle", 0)].toString());
```

Comment:
Can we make this simpler with just `tr("AETITLE")` ?

Reply:
ok,i will add the translation files to the demo app.
I use tr at first, but it seems can't work well. I will try again

Reply:
after test,  I have found that tr() will set the context with wrong one, it will cause the translation file can not be used. so i still use QApplication::translate